managedtenants/bundles/addon_bundles.py

```diff
@@ -61,13 +75,56 @@ def _parse_operator_bundles(self, operator_dir):
                 version=path.name,
             )
             res.append(bundle)
+
+        if len(res) == 0:
+            raise AddonBundlesError(
+                f"invalid structure for {self.root_dir}:"
+                f" {operator_dir} contains zero bundles."
+            )
         return res
 
     def _get_bundle_operator_name(self, operator_dir):
         operator_name = operator_dir.name
         return self.addon_name if operator_name == "main" else operator_name
 
-    def get_latest_version(self):
+    def _parse_and_validate_config(self):
+        config_file = self.root_dir / "main" / "config.yaml"
+        try:
+            with open(config_file, "r", encoding="utf-8") as f:
+                config = yaml.safe_load(f)
+                self._validate_config(config)
+                return config
+
+        except FileNotFoundError:
+            # We are not enforcing the presence of config.yaml yet.
```

Comment:
This is not good. Forgot to change that back. We want to enforce the `config.yaml` workflow (pushed a fix).